src/sentry/tagstore/snuba/backend.py

```diff
@@ -389,105 +417,8 @@ def get_groups_user_counts(self, project_id, group_ids, environment_id):
         aggregations = [['uniq', 'user_id', 'count']]
 
         result = snuba.query(start, end, ['issue'], None, filters, aggregations)
-        return defaultdict(int, result.items())
+        return defaultdict(int, {k: v for k, v in result.items() if v})
```

Comment:
This previously was returning `None` values which wouldn't fall through to the default 0 value and resulted in `NaN` showing up in the UI.